l10n_es_facturae/models/account_invoice.py

```diff
@@ -445,4 +463,56 @@ def _validate_facturae(self, xml_string):
                   "full error have been written in the server logs. Here "
                   "is the error, which may give you an idea on the cause "
                   "of the problem : %s") % str(e))
-        return True
+        return
```

Comment:
The true is missing

Reply:
No es realmente necesario.

Reply:
Igualmente mejor lo dejo igual a como estaba, no por que no afecte sinó por el diff